tokoh.md

```diff
@@ -3,5 +3,7 @@
 - David J Malan
 - Steve Jobs
 - Bill Gates
+- Budi Rahardjo
 - Ken C Doods
+- Jony Ive
```

Comment:
apakah anda yakin dia tokoh it ? 
sepertia dia designer

Reply:
baik, saya tidak akan ikutkan jony ive